README.md

```diff
@@ -97,7 +97,33 @@ Many browsers have started blocking cross-origin or "third party" cookies by def
   * [token.getWithoutPrompt](#tokengetwithoutpromptoptions) must have access to cookies on the Okta domain via an iFrame running on your application's page.
   * [token.renew](#tokenrenewtokentorenew) uses [token.getWithoutPrompt](#tokengetwithoutpromptoptions) and is subject to the same limitations.
 
-If your application depends on any of these methods, you should try to either rewrite your application to avoid using these methods or communicate to your users that they must enable third party cookies. Okta engineers are currently working on a better long-term solution to this problem.
+Notably, a browser blocking 3rd party cookies prevents automatic token renewal from working.   The code will behave as if the user's session at the issuer site has expired.
```

Comment:
it will break token renewal, automatic or otherwise. Maybe just remove the word "automatic".